### ISSUE TITLE
Users with special characters in their emails are sometimes blocked by Akismet during add-on submission

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-server#9501]
STR:

1. Create a new account using an email address that has some special characters (see the Notes for examples)
2. Submit a new add-on with this user - make sure that the add-on doesn't contain any potential spam content
3. Check the validation results

Actual result:
The submission is flagged by Aksimet for spam content

Expected result:
The submission should pass validation when the metadata is correct 

Notes:
- this may be be quite hard to reproduce, since there could be other conditions involved
- I'm adding here the accounts that had this problem: [user1](https://addons-internal.dev.mozaws.net/en-US/admin/models/users/userprofile/10641792/change/), [user2](https://addons-internal.dev.mozaws.net/en-US/admin/models/users/userprofile/10641780/change/). For comparison, [this user](https://addons-internal.dev.mozaws.net/en-US/admin/models/users/userprofile/10641755/change/) can enter any spammy content and his submissions/edits will pass without any errors all the time. 
Probably worth mentioning that all the above emails are deriving from the same gmail address.

**Comment by eviljeff** [comment]
can you provide samples of email address formats that consistently fail?

**Comment by AlexandraMoga** [comment]
I've added in my notes the links to those accounts in admin/users, but I guess I can simply add the emails here:
accounts that generate spam all the time:
g.aleksandra1102@gmail.com
galeksandra.1102@gmail.com


account that doesn't generate spam (ever):
galeksandra1102@googlemail.com

**Comment by eviljeff** [comment]
I tried to replicate this both with variants of my @mozilla.com address and with a new gmail address too (galeksandra.1234@gmail.com - almost the same as the ones you found consistently get flagged as spammy) but can't.  
It's impossible to get feedback from Akismet about why something failed so it must be something about the combination of the email, name, ip and metadata content that triggers it - it's not just a `.` in the email address.

We'll find out when enabled on prod how widespread this issue is... really the only fix for it is not sending the email address to Akismet (which will degrade it's accuracy in determining if the content is spammy).  Closing in the meantime.
